lib/util.js

```diff
@@ -408,8 +408,10 @@ module.exports.makeRemoteUrl = function(destination, req, serverConfig) {
   let zoweExternalPort;
   
   if(destination.includes('ZOWE_EXTERNAL_HOST') || destination.includes('ZWE_EXTERNAL_HOST')) {
-    if( referer > '') {
+    if( referer > '' && referer.includes(':')) {
       zoweExternalHost = referer.split(':')[1].substring(2); 
```

Comment:
Would suggest to use more reliable URL parser like this https://nodejs.org/docs/latest-v12.x/api/url.html#url_url_host to extract hostname from referrer.